src/__tests__/helpers/user-agents.json

```diff
@@ -33,10 +33,13 @@
   "iPodWebview": "Mozilla/5.0 (iPod; CPU OS 5_1 like Mac OS X) AppleWebKit/534.46 (KHTML, like Gecko) Mobile/3A101a",
   "iPhone_3_2Safari": "Mozilla/5.0 (iPhone; U; CPU iPhone OS 3_2 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Version/4.0.4 Mobile/7B314 Safari/531.21.10",
   "iPhone_9_3_1Safari": "Mozilla/5.0 (iPhone; CPU iPhone OS 9_3 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13E230 Safari/601.1",
+  "iPhone_15_5Safari": "Mozilla/5.0 (iPhone; CPU iPhone OS 15_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/103.0.5060.63 Mobile/15E148 Safari/604.1",
   "iPhoneUnsupportedChrome": "Mozilla/5.0 (iPhone; U; CPU iPhone OS 5_1_1 like Mac OS X; en) AppleWebKit/534.46.0 (KHTML, like Gecko) CriOS/47.0.1084.60 Mobile/9B206 Safari/7534.48.3",
   "iPhoneSupportedChrome": "Mozilla/5.0 (iPhone; U; CPU iPhone OS 5_1_1 like Mac OS X; en) AppleWebKit/534.46.0 (KHTML, like Gecko) CriOS/48.0.1084.60 Mobile/9B206 Safari/7534.48.3",
   "iPhoneFirefox": "Mozilla/5.0 (iPhone; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4",
   "iPhoneWebview": "Mozilla/5.0 (iPhone; U; CPU iPhone OS 4_1 like Mac OS X; en-us) AppleWebKit/532.9 (KHTML, like Gecko) Mobile/8B117",
+  "iPhone_15_5_Webview": "Mozilla/5.0 (iPhone; CPU iPhone OS 15 5 like Mac OS X) AppleWebKit/605.1.15(KHTML like Gecko) Version/15.5 Mobile/15E148 Safari/604.1",
```

Comment:
This we captured today from a Slack webview on an iPhone, so this is a very up to date user-agent. This scenario represents the case most in-need of resolving.

Reply:
So, currently, our solution is at odds with old iPads (`iPhone_3_2Safari`) and modern Webviews.